docs/assets/styles.css

```diff
@@ -386,6 +385,20 @@ section.cover .cover-main {
     max-width: 100px;
     max-height: 100px;
   }
+
+  body .sidebar-toggle { background-color: #fff; }
+  body.close .sidebar-toggle {
+    background-color: transparent;
+    padding: 1rem;
```

Comment:
are you sure you need this line? I see the padding declared also in `body .sidebar-toggle {}` and nothing overrides it (or maybe I am missing something).

Reply:
I need it because these styles are overriding another 3rd party CSS file :/